bulk-ingester/src/integration-test/java/org/alfresco/hxi_connector/bulk_ingester/processor/BulkIngestionFilterIntegrationTest.java

```diff
@@ -0,0 +1,90 @@
+
+package org.alfresco.hxi_connector.bulk_ingester.processor;
+
+import static org.alfresco.hxi_connector.bulk_ingester.util.IngestEventPropertyParser.parseProperties;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.annotation.Import;
+
+import org.alfresco.hxi_connector.bulk_ingester.BulkIngesterApplication;
+import org.alfresco.hxi_connector.bulk_ingester.util.integration.PostgresIntegrationTestBase;
+import org.alfresco.hxi_connector.common.model.ingest.IngestEvent;
+
+@EnableAutoConfiguration
+@SpringBootTest(properties = {"logging.level.org.alfresco=DEBUG",
+        "alfresco.filter.aspect.allow[0]=http://www.alfresco.org/model/content/1.0:auditable",
+        "alfresco.filter.aspect.deny[0]=http://www.alfresco.org/model/content/1.0:author"},
+        classes = BulkIngesterApplication.class)
+@Import(MockEventPublisherConfiguration.class)
+@SuppressWarnings({"PMD.TestClassWithoutTestCases", "PMD.SimplifyBooleanReturns", "PMD.LooseCoupling"})
+class BulkIngestionFilterIntegrationTest extends PostgresIntegrationTestBase
+{
+
+    @Autowired
+    private BulkIngestionProcessor bulkIngestionProcessor;
+
+    @Autowired
+    private DummyIngestEventPublisher ingestEventPublisher;
+
+    @Test
+    void shouldFilterOutNodeWithDeniedAspect()
+    {
+        // given
+        IngestEvent category = IngestEvent.builder()
+                .nodeId("94e0b276-6447-4dbc-b32a-1d37836a8066")
+                .properties(parseProperties("type=cm:category", "cm:name=Animal Species", "createdAt=1708329410", "aspectsNames=[cm:auditable]", "createdBy=admin", "modifiedBy=admin"))
+                .build();
+        IngestEvent folder = IngestEvent.builder()
+                .nodeId("dad275aa-affc-487d-a7ed-92cf8e6ce351")
+                .properties(parseProperties("type=cm:folder", "cm:name=Animals", "cm:title=Animals overview", "createdAt=1708329323", "aspectsNames=[cm:titled, cm:auditable]", "createdBy=admin", "cm:description=This folder contains overview of the animals in our zoo", "modifiedBy=admin"))
+                .build();
+        IngestEvent textFile = IngestEvent.builder()
+                .nodeId("44545a62-0f64-4d3e-838a-9f8ba23df0c7")
+                .contentInfo(new IngestEvent.ContentInfo(28, "UTF-8", "text/plain"))
+                .properties(parseProperties("cm:name=Animals list", "cm:title=List of animals", "type=cm:content", "createdAt=1708329628", "app:editInline=true", "aspectsNames=[cm:generalclassifiable, app:inlineeditable, cm:thumbnailModification, cm:titled, cm:taggable, rn:renditioned, cm:auditable]", "cm:categories=25805c3b-dc41-4a3d-9b03-a44ac7963c70", "createdBy=admin", "cm:lastThumbnailModification=[doclib:1708329633941, pdf:1708329629285]", "cm:description=List of animals in our zoo", "modifiedBy=admin"))
```

Comment:
I suggest removing `auditable` from one of these three nodes so that we test it's excluded.

Reply:
I thought I could do something like that but it appears it's not that easy.
We are preloading alfresco database with a dump which is used by this and several more tests - I was lazy enough to . 
I'll try to find a node without any of `allowed` aspects and include that in the test.